frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -96,6 +96,11 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
     }
   };
 
+  const catalogModelCustomProps: CatalogModelCustomProps = getCatalogModelCustomProps(
+    mv.customProperties,
+  );
+  const modelDetailsUrl = getModelDetailsUrl(catalogModelCustomProps);
```

Comment:
Sorry for another nit - `modelDetailsUrl` and `getModelDetailsUrl` could be confusing since this is the MR page and there are model details pages in both catalog and registry. Can we call these `catalogModelDetailsUrl` / `getCatalogModelDetailsUrl`?